template.yaml

```diff
@@ -1134,6 +1175,38 @@ Resources:
             Type: SQS
             Destination: !GetAtt ImportCandidateExpansionHandlerDLQ.Arn
 
+  DeleteImportCandidateEventHandler:
+    DependsOn: EventsLambdaPermissions
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-event-handlers
+      Handler: no.unit.nva.publication.events.handlers.delete.DeleteImportCandidateEventHandler::handleRequest
+      Runtime: java11
+      Role: !GetAtt LambdaRole.Arn
+      ReservedConcurrentExecutions: 64
```

Comment:
Excessive?

Reply:
Should I change reserved concurrency for all lambdas in deletion flow, or only this one? It is the same amount as ExpansionHandler uses and this one does similar action, i.e. its sends event to searchApi.

Reply:
I think 1 should be enough. 
If we have to resort to 64 concurrent executions for deleting a bunch of scopus-identifiers, it's time to consider a sqs queue or simply dumping the database.